lib/payload.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schema.thethings.network/devicerepository/1/payload/schema",
+  "title": "LoRaWAN Device Repository Payload",
+  "description": "Payload definitions for the LoRaWAN Device Repository",
+  "definitions": {
+    "temperature": {
+      "type": "number",
+      "description": "Temperature (°C)",
+      "minimum": -273.15
+    },
+    "direction": {
+      "type": "number",
+      "description": "Direction (°)",
+      "minimum": 0,
+      "exclusiveMaximum": 360
+    },
+    "speed": {
+      "type": "number",
+      "description": "Speed (m/s)",
+      "minimum": 0
+    },
+    "measurement": {
+      "type": "object",
+      "properties": {
+        "time": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the measurement (RFC3339)"
+        },
+        "air": {
+          "type": "object",
+          "properties": {
+            "temperature": {
+              "description": "Air temperature (°C)",
+              "$ref": "#/definitions/temperature"
+            },
+            "relativeHumidity": {
+              "type": "number",
+              "description": "Relative humidity (fraction)",
+              "minimum": 0,
+              "maximum": 1
+            },
```

Comment:
```suggestion
            "relativeHumidity": {
              "description": "Relative humidity (fraction)",
              "$ref": "#/definitions/fraction"
            },
```

Reply:
Yeah I think this could come in handy indeed.

I went with [0..1] instead of [0..100] because the device repository already has a notion of relative humidity in fraction, namely the [operating conditions](https://github.com/TheThingsNetwork/lorawan-devices/blob/master/schema.json#L1036-L1053). Now, I extracted the schema to `lib/payload.json` also because these schemas likely develop independently from now on. So I'm also okay with [0..100]; percentages may be more common to express most dimensions I suppose?

Reply:
I like the separation in a new file, the scheme was starting to get pretty convoluted. As for fractions vs percentages, I don't have a strong opinion, but I think percentage is more appropriate almost everywhere and more useful for integrations to avoid further conversions.